x/auth/tx/gogotx_test.go

```diff
@@ -0,0 +1,26 @@
+package tx
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	v1beta1 "cosmossdk.io/api/cosmos/tx/v1beta1"
+	"cosmossdk.io/x/tx/decode"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	addresscodec "github.com/cosmos/cosmos-sdk/codec/address"
+	codectestutil "github.com/cosmos/cosmos-sdk/codec/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func TestNewWrapperFromDecodedTx(t *testing.T) {
```

Comment:
Can we make this test name more specific?
```suggestion
func TestNewWrapperFromDecodedTxAllowsNilFee(t *testing.T) {
```